docs/conf.py

```diff
@@ -40,7 +48,24 @@
 # -- Options for HTML output ----------------------------------------------
 
 # We want to set the RTD theme, but not if we're on RTD.
-if os.environ.get('READTHEDOCS', '') != 'True':
+if os.environ.get('READTHEDOCS', '') == 'True':
```

Comment:
Let's follow their example.

Just using os.getenv sounds like a bad idea in case the environment is ever populated with a "READTHEDOCS"/"False" key-value pair.